test/models/deploy_service_test.rb

```diff
@@ -160,6 +160,15 @@ class DeployServiceTest < ActiveSupport::TestCase
       job_execution.send(:run!)
     end
 
+    it "sends Zendesk notifications if the stage has them enabled" do
+      stage.stubs(:comment_on_zendesk_tickets?).returns(true)
+
+      ZendeskNotification.any_instance.expects(:deliver)
```

Comment:
hmm that test would have passed before with `:deliver_now`, any other sideeffect we could spec on ?